src/Views/Nav.elm

```diff
@@ -29,12 +29,6 @@ view model =
             , navItem (t IconContact) (t ContactLinkDestination) "contact" "email" (t ContactLinkShort) (t ContactLinkLong)
             , span [ class "nav-item mobile-only" ]
                 [ callToActionNavItemMobile (callToActionConstructor Survey (t CallToActionDestination) (t CallToActionDestinationDisplay)) ]
-            , span [ class "nav-item" ]
-                [ a [ class "btn ", href "https://google.com", target "_blank", rel "noopener", onClick Exit ]
-                    [ getIcon (t IconExit) (Just "nav-item-text nav-icon")
```

Comment:
@Nikomus Do you think it is worth removing the Exit icon from the code also? Or not worth the hassle?

Reply:
To be honest I think it's not worth the hassle but it's much better to have clean code, so I'll update.

Reply:
No problem. If we don;t do it now, we never will and it's not too ugly but a little sad. Let's wait and see what @Nikomus thinks.

Reply:
There's a brand copy key of `IconExit`, which if this is used as a template might be confusing? The `.icon-exit-door:before` selector in icons.css could go too, though I wouldn't worry about rebuilding a new icon font. New partners will get a new font anyway.

Reply:
We'll be sync-ing the keys with BrandCopy.elm in hospice shortly.

Good point about the rebuild of icon font.

@RabidGamer just the code to remove, don't worry about the font.